shim.js

```diff
@@ -5809,22 +5809,22 @@ dnode.prototype = {};
 function dnode (cons, opts) {
```

Comment:
On compilation of shim.coffee (coffee -c shim.coffee)  the resulting shim.js file is significantly smaller than the current shim.js in the master branch.  I took the compiled part:

onPageCreated: function(cb) {
        if (cb == null) {
          cb = (function() {});
        }
        return page.onPageCreated = function(newpage) {
          return cb(pageWrap(newpage));
        };
      },

 from the resulting shim.js file and pasted it into the pageWrap function leaving the rest of the shim.js file unchanged when compared to the master copy.  On commit the only piece flagged as changed was the above code snippet.  Let me know if you have any questions.


On Sep 16, 2015, at 2:10 PM, Amir Raminfar <notifications@github.com<mailto:notifications@github.com>> wrote:


In shim.js<https://github.com/sgentle/phantomjs-node/pull/310#discussion_r39664911>:

> @@ -5809,22 +5809,22 @@ dnode.prototype = {};
>  function dnode (cons, opts) {


Did you recompile the js file? I don't get why the extra lines are showing up. They shouldn't be there if you compiled it.

—
Reply to this email directly or view it on GitHub<https://github.com/sgentle/phantomjs-node/pull/310/files#r39664911>.